demo/License/LicenseViewController.swift

```diff
@@ -6,20 +6,26 @@ protocol LicenseDelegate: class {
     func licenseSubmitted()
 }
 
-final class WelcomeViewController: UIViewController {
+final class LicenseViewController: UIViewController {
     weak var licenseDelegate: LicenseDelegate?
-    
-    private let licensePreviewController = LicenseController.previewController()
+
+    init(buttonTitle: String = L10n.welcomeButton, title: String = L10n.welcomeTitle) {
```

Comment:
No, it was used when I thought that the screen can be used in two places (in license screen at the start and in info screen from menu), for now it's unused, it can be removed and I see no need to do it, but if you think so, I'll remove

Reply:
done